projects/kit/directives/highlight/highlight.options.ts

```diff
@@ -0,0 +1,10 @@
+import {tuiCreateOptions} from '@taiga-ui/cdk/utils/di';
+
+export interface TuiHighlightOptions {
```

Comment:
No, I mean drop that interface, because

<img width="1121" alt="image" src="https://github.com/user-attachments/assets/dd8b8988-43f9-4cbb-b0b6-619847586c13">

Reply:
[e22f982](https://github.com/taiga-family/taiga-ui/pull/8812/commits/e22f982215435e478bc2b2b3f964cb398b5dc4a5)